### PR TITLE
Remove `env` command option support

### PR DESCRIPTION
**Pull request by jainvikas8**
### Description

The `env` option just displays help information as opposed to reading
environment variables or configuring them.

It's better this information is reflected in docs rather than on
command options.
 
### Test Coverage

- [x ]  This change is covered by existing or additional automated tests.
- [ ]  Manual testing has been performed (and evidence provided) as automated testing was not feasible.
- [ ]  Additional tests are not required for this change (e.g. documentation update).

### CODE REVIEW

File: src/mbed_tools/cli/env_cli.py [hunk 1]
```diff
@@ -1,42 +0,0 @@
...
-from mbed_tools.targets.env import env_variables as devices_env_variables
-
-
-@click.command()
-def cli() -> None:
-    """Prints information about environment variables of all child mbed packages."""
-    click.echo(_build_output(devices_env_variables))
-
-
-_OUTPUT_PREAMBLE = """These variables used by Mbed Tools can be set either directly via environment variables or
```
Comment (Patater) [inline src/mbed_tools/cli/env_cli.py:19]: Where can I find this OUTPUT_PREAMBLE stuff in docs now?

**Review by Patater — COMMENTED** [review]
Do we need to delete anything from the docs/api folder (perhaps by running the generate docs command to do so)?

File: src/mbed_tools/cli/env_cli.py (hunk 1 above)
Comment (jainvikas8) [reply src/mbed_tools/cli/env_cli.py:19]: We can find it here - https://github.com/ARMmbed/mbed-tools/pull/75/files#diff-a650c08c2a3b6de7e502c89ee35ef1aeR129

**Comment by jainvikas8** [comment]
> Do we need to delete anything from the docs/api folder (perhaps by running the generate docs command to do so)?

Will, it not get auto-generated? I see its python code.

**Comment by jainvikas8** [comment]
Forced pushed to remove html files from docs/api

**Comment by jainvikas8** [comment]
Closing this PR due to CI not able to checkout a forked branch.

**Pull request closed without merging**